### DISCUSSION TITLE
[RELEASE] Area 2-13 (1.10)

### DISCUSSION THREAD

**Discussion by DerpedCrusader** [nzp-team/nzportable#308] [Map Releases]
Long in the works, the half life themed map finally releases

its a decently "large" sized map, in NZP terms

i tried to make sure it has everything

- all perks
- pap
- teleporter
- traps

# Screenshots
<img width="1578" height="870" alt="image" src="https://github.com/user-attachments/assets/396d71a9-bf8f-4787-8d7b-0867f9a3a62a" />
<img width="1580" height="864" alt="image" src="https://github.com/user-attachments/assets/6a5d4778-44de-440a-9a9f-33229f46bb6f" />
<img width="1592" height="848" alt="image" src="https://github.com/user-attachments/assets/172dbdc7-b51f-46e4-9461-65c6a4fb90a2" />

![fte-20230704110703-0](https://github.com/nzp-team/nzportable/assets/13917665/6f7edd85-0ce9-4005-9797-e76c829b4777)

# [Download](https://archive.org/download/area2-13_202404/area_2-13_1.10.zip)

[Source files (requires Halflife.wad from commercial copy of HL1) ](https://archive.org/download/area2-13_202404/HalfCrapUncut1.10.map)

**Comment by MotoLegacy** [comment]
neat

**Comment by MotoLegacy** [comment]
![fte-20230707163227-0](https://github.com/nzp-team/nzportable/assets/26030250/053e3147-3322-4766-8b48-34fb7025bcf5)
Looks like this trigger is set up incorrectly

**Comment by blubs** [comment]
10/10 would play again

**Comment by DerpedCrusader** [comment]
1.01 patch notes

Fixed a way point issue

**Comment by G00berF4n** [comment]
Besides some pathing issues and the mp5 map, I like how short and action packed this map can be, my only wish was that spare change (getting 100 points from going prone infront of a perk machine) was a thing.

**Reply by DerpedCrusader** [reply]
Where are the pathing issues? could you send screenshots?

**Comment by sealteamaqua** [comment]
i really need .bsp files in order for it to work.... older posts had those

**Reply by Revnova** [reply]
are you clicking the blue button at the bottom that says "Download"?

**Reply by Revnova** [reply]
the text that says "Download" is the literal download button. its the same for most maps posted on here

**Comment by DerpedCrusader** [comment]
update version 1.10!

fixed some waypoints, decorated some areas a bit more, added ZONING SO THE MAP IS ACTUALLY FUUUUNNNNNN!!!!!!!!!!!!!!!!!!!!!!!

some areas are still way too open, but i really wanted zones to be out for people

updated the link in the main post

<img width="1568" height="822" alt="image" src="https://github.com/user-attachments/assets/599ebe11-f5c8-4e03-a092-89ab54da69d1" />
<img width="1594" height="836" alt="image" src="https://github.com/user-attachments/assets/510db95e-fc9f-4a50-9608-66c8b01f161d" />

**Reply by ImTander** [reply]
Such a official looking map! Great job Derp!

**Comment by josephgraymobile-spec** [comment]
Crashes on the vita port when loading, am i missing a file? I just used the download button.
